server/routes/calendar.js

```diff
@@ -16,17 +16,14 @@ router.get(
   '/:date',
   asyncMiddleware(async (req, res) => {
     logger.info('GET calendar view')
-    const {
-      calendarService,
-      calendarOvertimeService,
-      DEPRECATEnotificationService,
-      userAuthenticationService,
-    } = req.app.get('DataServices')
+    const { calendarService, calendarOvertimeService, notificationService, userAuthenticationService } = req.app.get(
+      'DataServices',
+    )
 
     try {
       const userAuthenticationDetails = await userAuthenticationService.getUserAuthenticationDetails(req.user.username)
 
-      const shiftNotifications = await DEPRECATEnotificationService.getShiftNotifications(req.user.username)
+      const notificationCount = await notificationService.countUnprocessedNotifications(req.user.token)
```

Comment:
Pet hate again. `req.some.parameter.that.gets.accessed.multiple.times.could.be.destructed`
I just feel it makes the code more readable 🤷‍♀️

Reply:
I think this is a personal preference thing. I often prefer to have the service/model name when using methods as it makes it clear what I am connecting to. I find orphaned method names trickier to conceptualise how the app fits together

Reply:
A quick glance, I don't think the `req` params require context? Token, username, employeeName etc.
For the record though, I find `countUnprocessedNotifications` descriptive enough to stand alone too 🤔

Reply:
Ahh your comment was on req rather than the method. I can pull those off.